File: services/python/flask/src/routes/handlers/invalid_path.py
import structlog
from flask import request
from werkzeug.exceptions import HTTPException

from routes.errors import AppErrors, send_error


def invalid_path_handler(_error: HTTPException):
    structlog.get_logger().warn(
        "Invalid path", method=request.method, path=request.path
    )
    return send_error(AppErrors.InvalidPath)

File: services/go/chi/routes/handlers/invalid_path.go
package handlers

import (
	"chi/routes"
	"log/slog"
	"net/http"
)

func InvalidPathHandler(w http.ResponseWriter, r *http.Request) {
	slog.Warn("Invalid path", "method", r.Method, "path", r.URL.Path)

	routes.AppErrors.InvalidPath.Error(w)
}
